web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -107,43 +105,33 @@ export interface Gateway extends apiGateway.Gateway.AsObject {
  * if the user was to invoke it manually from the terminal. Should not be actually used to execute
  * anything in the shell.
  */
-export type GatewayCLICommand = apiGateway.GatewayCLICommand.AsObject;
+export type GatewayCLICommand = apiGateway.GatewayCLICommand;
 
-export type AccessRequest = apiAccessRequest.AccessRequest.AsObject;
-export type ResourceId = apiAccessRequest.ResourceID.AsObject;
-export type AccessRequestReview = apiAccessRequest.AccessRequestReview.AsObject;
-export type AccessList = apiAccessList.AccessList.AsObject;
+export type AccessRequest = apiAccessRequest.AccessRequest;
+export type ResourceId = apiAccessRequest.ResourceID;
+export type AccessRequestReview = apiAccessRequest.AccessRequestReview;
+export type AccessList = apiAccessList.AccessList;
 
-export interface GetServersResponse
-  extends apiService.GetServersResponse.AsObject {
-  agentsList: Server[];
+export interface GetServersResponse extends apiService.GetServersResponse {
+  agents: Server[];
 }
 
-export interface GetDatabasesResponse
-  extends apiService.GetDatabasesResponse.AsObject {
-  agentsList: Database[];
+export interface GetDatabasesResponse extends apiService.GetDatabasesResponse {
+  agents: Database[];
 }
 
-export interface GetKubesResponse extends apiService.GetKubesResponse.AsObject {
-  agentsList: Kube[];
+export interface GetKubesResponse extends apiService.GetKubesResponse {
+  agents: Kube[];
 }
 
-export interface GetAppsResponse extends apiService.GetAppsResponse.AsObject {
-  agentsList: App[];
+export interface GetAppsResponse extends apiService.GetAppsResponse {
+  agents: App[];
 }
 
 export type GetRequestableRolesResponse =
-  apiService.GetRequestableRolesResponse.AsObject;
-
-export type ReportUsageEventRequest = Modify<
-  apiUsageEvents.ReportUsageEventRequest.AsObject,
-  {
-    prehogReq: Modify<
-      apiUsageEvents.ReportUsageEventRequest.AsObject['prehogReq'],
-      { timestamp: Date }
-    >;
-  }
->;
+  apiService.GetRequestableRolesResponse;
+
+export type ReportUsageEventRequest = apiUsageEvents.ReportUsageEventRequest;
```

Comment:
This change should be fine I think, @gzdunek AFAIK it was to accommodate for some weird Timestamp shenanigans in protos?

Reply:
I think so, I will make sure that the correct date is sent with the usage events.

Reply:
Except for the wrong environment, usage events are sent correctly.